for-comments.ino

```diff
@@ -0,0 +1,374 @@
+    digitalWrite(redPin, LOW);// turn LED off, added
+    delay(500);
+    digitalWrite(redPin, HIGH);// turn LED off, added
+    delay(500);
+    digitalWrite(redPin, LOW);// turn LED off, added
+    delay(500);
+    asm volatile ("  jmp 0"); // invoke restart of arduino
+  }
+}
+/////End of Function////
+
+
+
+///take weight reading///
+void GetWeight()
+{
+  RawValue();
+  
+  Serial.println("Weighing Mode Active"); // added
+  ForceTemp = ((LoadB - 0.0) / (ValueB - ValueA)) * ((ForceValueRaw) - ValueA) + 0.0;
+  ValueOut = ForceTemp;
+
+  
+  Serial.print("ForceValueRaw> ");
+  Serial.print(ForceValueRaw);
+  Serial.print(" kilograms> ");
+  Serial.print(ValueOut);
+  Serial.print(" LoadB> ");
+  Serial.print(LoadB);
+  Serial.print(" ValueA> ");
+  Serial.print(ValueA);
+  Serial.print(" ValueB> ");
+  Serial.print(ValueB);
+  Serial.print(" PC>");
+  Serial.println(CounterPulse);
+  delay(500);
+
+  ConnectNetwork();
+  delay(500);
+  
+  Send2Xively();
+}
+
+/////End of Function////
+
+
+
+///Send To Xively Function///
+void Send2Xively()
+{
+  /*
+  GPRS.println("AT+CGATT=1"); // Attach from GPRS serivce
+  Response();
+  delay(1000);
+  GPRS.println("AT+CSTT=\"internet\""); // Start task and set APN
+  delay(3000);
+  Response();
+  GPRS.println("AT+CIICR"); // Bring up wireless connection with GPRS
+  delay(2000);
+  Response();
+  GPRS.println("AT+CIFSR"); // Get local IP address
+  Response();
+  GPRS.println("AT+CIPSPRT=1"); // Set its prompt echo '>'and shows "send ok" when sending is successful
+  Response();
+  GPRS.println("AT+CIPQSEND=0"); // Set normal data transmitting mode
+  Response();
+*/
+  
+  GPRS.println("AT+CIPSTART=\"tcp\",\"api.xively.com\",\"8081\""); //Start up TCP connection
+  delay(2000);
+  Response();
+  GPRS.println("AT+CIPSEND"); // Begin send data to Xively through TCP connection
+  Response();
+  // Request Syntax for TCP Socket Connection Type
+  GPRS.print("{");
+  GPRS.print("\"method\" : \"put\",");
+  GPRS.print("\"resource\" : \"/feeds/");
+  GPRS.print(FEEDID);
+  GPRS.print("\",");
+  GPRS.print("\"params\" : {},");
+  GPRS.print("\"headers\" : {\"X-ApiKey\":\"");
+  GPRS.print(APIKEY);
+  GPRS.print("\"},");
+  GPRS.print("\"body\" : ");
+  GPRS.print("{\"version\" : \"1.0.0\",");
+  GPRS.print("\"datastreams\" : ");
+//  GPRS.print("[{\"id\" : \"weight\",");
+  GPRS.print("[{\"id\" : \"");
+  GPRS.print(SCALEID);
+  GPRS.print("\",");
+  GPRS.print("\"current_value\" : \"");
+  GPRS.print(ValueOut,1);
+  GPRS.println("\"}]},"); 
+  GPRS.print("\"token\" : \"lee\"}");
+  delay(500);
+  // End of Request Syntax
+  GPRS.println((char)26); // After sending AT+CIPSEND ,tap CTRL+Z to send data
+  delay(500);
+  Response();
+//  GPRS.println("AT+CIPCLOSE"); // Close TCP connection
+//  Response();
+//  delay(3000);
+CloseConnection();
+}
+
+void Response()
+{
+  delay(100);
+  while(!GPRS.available());
+  if(GPRS.available()){
+    while(GPRS.available()){
+      Serial.write(GPRS.read());
```

Comment:
I think I have merged the branch add feature/rtcsync with the master of my
copy file.. cant seem to delete that branch (I hadn't understood the copies
being on Github online at the time, I still thought we were linked to just
1 file).. Essentially though I just want to start fresh, i.e. my "final"
master (which I will only edit once I have tested in copy file) and my
"copy" master, No branches. How do I take it back to zero?? I tried just
removing the repository on Gitdesktop but when I added it again the
branches were still there, maybe from Gitonline??

On Fri, Jun 24, 2016 at 9:39 PM, William Stewart <
william@silverfernhoney.co.nz> wrote:

> should I set up the code Im working on within my add feature branch??? so
> just get rid of current code in my test file and replace with this code??
>
> On Fri, Jun 24, 2016 at 9:13 PM, fawdg <notifications@github.com> wrote:
>
>> In for-comments.ino
>> <https://github.com/NZbeek/Scales---Sim900_v3.0/pull/2#discussion_r68372490>
>> :
>>
>> > +  GPRS.println((char)26); // After sending AT+CIPSEND ,tap CTRL+Z to send data
>> > +  delay(500);
>> > +  Response();
>> > +//  GPRS.println("AT+CIPCLOSE"); // Close TCP connection
>> > +//  Response();
>> > +//  delay(3000);
>> > +CloseConnection();
>> > +}
>> > +
>> > +void Response()
>> > +{
>> > +  delay(100);
>> > +  while(!GPRS.available());
>> > +  if(GPRS.available()){
>> > +    while(GPRS.available()){
>> > +      Serial.write(GPRS.read());
>>
>> Do you know what Serial.write(); does? Is it just printing the output of
>> GPRS.read()?
>>
>> —
>> You are receiving this because you are subscribed to this thread.
>> Reply to this email directly, view it on GitHub
>> <https://github.com/NZbeek/Scales---Sim900_v3.0/pull/2/files/ca8d53f7707eb5dace508baab47a998f8759ff3b#r68372490>,
>> or mute the thread
>> <https://github.com/notifications/unsubscribe/AHxY5bQPNIhylivPL6oyAZXz0dTKajX0ks5qO5_KgaJpZM4I9l8d>
>> .
>>
>
>